app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id='error_explanation'>
+    <div class='alert alert-danger'>
+      The form contains <%= pluralize(@user.errors.count, 'error') %>.
+    </div>
+    <ul>
+    <% @user.errors.full_messages.each do |msg| %>
```

Comment:
is it displayed correctly on the view? as far as I know, the `li` tag should be inside in `ul`